tests/testing/chdir/chdir.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"log"
+	"os"
+	"runtime"
+	"strings"
+)
+
+/*
+Test that this program is 'run' in expected directory. 'run' with expected
+working-directory in 'EXPECT_DIR' environment variable' with{,out} a -C
+argument.
+*/
+func main() {
+	expectDir := os.Getenv("EXPECT_DIR")
+	cwd, err := os.Getwd()
+	if err != nil {
```

Comment:
Note this test failed on Windows CI, so we might need more here.

Reply:
Fixed, thanks for the review!